testsuite/tests/tool-ocamldoc-html/Makefile

```diff
@@ -33,10 +33,11 @@ default:
 	fi
 
 .PHONY: run
-run: *.mli
-	@for file in *.mli; do \
+run: *.mli *.ml
+	@for file in *.ml *.mli; do \
 	  printf " ... testing '$$file'"; \
 	  F="`basename $$file .mli`"; \
+	  F="`basename $$F .ml`"; \
```

Comment:
So in bash `${F%.*}` works to strip the extension, but I'm fine with this instead which might be more portable (?). Maybe having a comment (`# stripping both .ml and .mli extensions`) could help people less familiar with this use of `basename`.